README.md

```diff
@@ -79,6 +79,28 @@ The server supports the following queries:
 
 Note that degrees of support for each LSP request are varying.
 
+## Contribution
+
+```bash
+# clone repo with submodules
+git clone --recursive git@github.com:ocaml/ocaml-lsp.git
+
+# if you already clone, pull submodules
+git submodule update --init --recursive
+
+# creat local switch (or use global one)
+opam sw create . 4.09.0
```

Comment:
I tried reproducing the steps, but I got this
```
opam sw create . 4.09.0
[ERROR] Compiler selection '4.09.0' is ambiguous. matching packages: { ocaml-base-compiler.4.09.0, ocaml-system.4.09.0 }
```
It depends on which system compiler you have, but let's replace this to make it explicit ?
Also, we might as well go with 4.09.1
```suggestion
opam switch create . ocaml-base-compiler.4.09.1
```

Reply:
Oh, I missed it. (I deleted my suggestion about this to avoid confusion.)